schemas/amulet/20210221.json

```diff
@@ -0,0 +1,69 @@
+{
+    "description": "This schema describes version 20210221 of the metadata standard for the amulet poem format, designed for the Zora protocol.",
+    "title": "Amulet20210221",
+    "$id": "http://text.bargains/schema.json",
+    "type": "object",
+    "properties": {
+        "description": {
+            "description": "The description of the amulet, which should include at minimum (1) a link to the current formal definition and (2) a link to the carbon offset purchased in the poem's name.",
```

Comment:
Any reason why the carbon offset is not its own key / value pair in the schema?

Reply:
No reason at all, and that's a great idea! Help me think through one part of it: obviously, I am inclined to make the field required, but, the first amulets (which… "in theory"… follow this schema) don't have it. Will that cause problems? Or is this really just about validating metadata going forward, so it's fine?

Reply:
What if you leave `amulet-20210221` as it is, so your first amulets validate against it, and then create `amulet-20210227` for future amulets? I think this is probably overkill (how exacting are NFT people?) but offering it for discussion anyway.

Reply:
Yeah, I had the same thought, then felt bashful about taking so much "space" with such a simple project 🤔

But I appreciate the nudge…

Reply:
if you want to be backwards compatible with the existing amulets then you're right you should probably keep it as is, and consider making a later versioned schema to improve it, if folks deem it to be neccessary.

Reply:
That sounds good to me -- I've removed the 20210228 version and we'll stick with some optional fields in 20210221 for now. Thanks for the input!